charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: v1
+data:
+  aws-ccm-config: |
```

nit: IMO the "config" here seems unnecessary since its being used as ` n.helmAddonConfigGetter.GetSettingsFor()`
```suggestion
  aws-ccm: |
```

---

removing.